include/verilated_funcs.h

```diff
@@ -2339,6 +2339,169 @@ inline IData VL_VALUEPLUSARGS_INQ(int rbits, const std::string& ld, double& rdr)
 }
 extern IData VL_VALUEPLUSARGS_INN(int, const std::string& ld, std::string& rdr) VL_MT_SAFE;
 
+//======================================================================
+// bitHelper
+
+template <typename OutWordT, typename InWordT>
+void alignedBitHelper2(OutWordT* outWords, uint32_t outLsB, const InWordT* inWords, uint32_t inLsB,
+                       uint32_t width) {
```

Comment:
Please rename indicating what it does.  Can we avoid the template?  Can the several functions be merged in some way?

Reply:
Have pushed a more simplified implementation